listenbrainz/webserver/views/user.py

```diff
@@ -349,6 +350,35 @@ def collaborations(user_name: str):
     )
 
 
+@user_bp.route("/<user_name>/pins/")
```

Comment:
Need to add methods and CORS header like https://github.com/metabrainz/listenbrainz-server/pull/1588